test/unit/sync/test_sync_report.py

```diff
@@ -10,12 +10,10 @@
 
 from unittest.mock import MagicMock
 
-from ..test_base import TestBase
+from apiver_deps import SyncReport
 
-from .deps import SyncReport
 
-
-class TestSyncReport(TestBase):
+class TestSyncReport:
```

Comment:
it no longer inherits from TestBase!

Reply:
correct. It was moved to the new test structure, which uses pytest and no longer requires test classes to inherit from TestBase. In fact, it sometimes hurts when they do.